src/sync/installation.ts

```diff
@@ -33,16 +27,19 @@ interface TaskProcessors {
 		(
 			github: GitHubAPI,
 			repository: Repository,
-			cursor: string | number,
-			perPage: number
+			cursor?: string | number,
+			perPage?: number
 		) => Promise<{ edges: any[], jiraPayload: any }>;
 }
 
 type TaskType = "pull" | "commit" | "branch";
 
 const taskTypes = Object.keys(tasks) as TaskType[];
 
-const completedTaskStatuses : TaskStatus[] = ["complete", "failed"];
+// TODO: why are we ignoring failed status as completed?
```

Comment:
That's a change made recently by Tom. We do it to fix the case where we fail the whole sync if one repo fails. We count it as completed so we can continue through the rest of repositories (hence we are not going to do anything with the failed repo again)